src/test/java/com/homework/lesson11/FruitsShop.java

```diff
@@ -0,0 +1,83 @@
+package com.homework.lesson11;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.NoArgsConstructor;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+@NoArgsConstructor
+public class FruitsShop {
+    private List<Fruits> fruitsArrayList = new ArrayList<>();
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    public void addFruits(String pathToJsonFile) throws IOException {
+        List<Fruits> fruitsList;
```

Comment:
better write like this `        List<Fruits> fruitsList = (objectMapper.readValue(new File(pathToJsonFile),
 new TypeReference<List<Fruits>>() { }));`

Reply:
I mean dont separate declaring from initialization